.github/workflows/snyk-sbom.yml

```diff
@@ -27,7 +31,7 @@ jobs:
 
       - name: sbom/generate-monorepo
         if: ${{ inputs.is_monorepo }}
-        run: snyk sbom --format=cyclonedx1.5+json --all-projects > sbom.json
+        run: snyk sbom --format=cyclonedx1.5+json --all-projects > sbom-${{ inputs.repo_name}}}-${{ github.event.release.tag_name }}.json
```

Comment:
I am not sure this would work: the trigger event is `workflow_call` for this workflow, and not `release`, so afaiu the `github.event.release` object should be empty

Reply:
It currently works. You can see it here when we do `gh release upload "${{ github.event.release.tag_name }}" sbom.json`

https://github.com/mattermost/mattermost/actions/runs/12271195180/job/34237656843

Reply:
I just noticed now while reviewing your other PR, but there seems to be a typo here:  `${{ inputs.repo_name}}}` should be `${{ inputs.repo_name}}` (there's one `}` too much). Same in the line below
cc @enzowritescode

Reply:
I also wonder if it's possible to retrieve the repository name from the `github.event` itself, instead of passing it as an input? e.g. uising `${{ github.event.repository.name }}`